src/main/java/uk/gov/hmcts/reform/pip/rules/repository/InMemoryRepository.java

```diff
@@ -71,5 +106,39 @@
     }
 
 
+    /**
+     * Method to retrieve a court hearings using court ID.
+     * @param courtId The ID of the court.
+     * @return The court hearings to return, if found.
+     */
+    public Optional<CourtHearings> getCourtHearings(Integer courtId)
+    {
+        CourtHearings court = this.courtHearings.stream()
```

Comment:
can we rename `CourtHearings` to `Court`, court hearings isnt right, those are the hearings

Reply:
renamed